config/initializers/content_security_policy.rb

```diff
@@ -0,0 +1,25 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
This is interesting, did the rails upgrade generate this file or did something else?

Reply:
Yes, rails update command generated it ... I usually delete the stuff in config/initializers that's not currently relevant to the project, but this one I suspect will become relevant post-6.0 and/or as apps start to lock down security better.